test/integration/getserverprops/pages/something.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Link from 'next/link'
+import { useRouter } from 'next/router'
+
+// eslint-disable-next-line camelcase
+export async function unstable_getServerProps({ params, query }) {
+  return {
```

Comment:
This should be nested under the `props` key for consistency with the other life cycles.

Reply:
Updated to nest under `props` and check for invalid keys